types/vimeo__player/index.d.ts

```diff
@@ -286,7 +299,7 @@ export interface EventMap {
 export type EventCallback<Data = any> = (data: Data) => any;
 
 export type VimeoTimeRange = [number, number];
-export type VimeoVideoQuality = '4K' | '2K' | '1080p' | '720p' | '540p' | '360p' | '240p';
+export type VimeoVideoQuality = 'auto' | '4K' | '2K' | '1080p' | '720p' | '540p' | '360p' | '240p';
```

Comment:
I've added `'auto'` here as it is also a valid video quality, and one that makes sense to have now that the quality methods have been added.